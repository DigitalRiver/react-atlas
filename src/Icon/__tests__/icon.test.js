import React from "react";
import { mount } from "enzyme";
import { Icon } from "../index";

import renderer from "react-test-renderer";

let icon = 'icon={<i className="fa fa-github"></i>}';

describe("Test correct render", () => {
  it("Test correct render", function() {
    const tree = renderer.create(<Icon icon={icon} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe("Testing Icon component", () => {
  it("Set props should match what was passed in", function() {
    const result = mount(<Icon icon={icon} />);
    expect(result.props().icon).toBe(icon);
  });
});
