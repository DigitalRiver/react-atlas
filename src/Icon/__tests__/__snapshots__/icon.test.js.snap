// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test correct render Test correct render 1`] = `
<i
  className="icon={<i className=\\"fa fa-github\\"></i>}"
  onClick={undefined}
  style={undefined}
/>
`;
